assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -65,56 +71,65 @@ const ChecklistTab = ({ checklist, currentCheckpoint }) => {
 
   const { highPriority, recommended, pages } = useMemo(
     () =>
-      checklist.reduce(
-        (prevMessages, current) => {
-          const id = uuidv4();
-          const currentMessage = { ...current, id };
-          const isHighPriority =
-            currentMessage.type === PRE_PUBLISH_MESSAGE_TYPES.ERROR;
-
-          const [typeKey, copyKey] = isHighPriority
-            ? ['highPriority', 'recommended']
-            : ['recommended', 'highPriority'];
-
-          if (currentMessage.page !== undefined) {
-            const prevTypeGroup = prevMessages.pages[typeKey] || {};
-            const prevPageGroup = prevTypeGroup[currentMessage.page] || [];
-
-            let prevLengths = { highPriority: 0, recommended: 0 };
-            if (prevMessages.pages?.lengths) {
-              ({ lengths: prevLengths } = prevMessages.pages);
-            }
+      checklist
+        // TODO remove filtering out video optimization check
```

Comment:
(mostly a note for myself, add a new issue for removing video optimization feature flag)

Reply:
https://github.com/google/web-stories-wp/issues/7322